db/python/tables/sample.py

```diff
@@ -49,7 +49,6 @@ class SampleTable(DbBase):
         'active',
         'type',
         'project',
-        'audit_log_id',
```

Comment:
An alternative would be to add an `audit_log_id` field to `SampleInternal`. However most other models don't bother having such a field, so I went with leaving it off and we can add it later in the event we ever want to manipulate it.

Reply:
Na this is pretty reasonable, I'll need to have a better think about audit logs, I think returning the "last edited author" is a bit silly in a way.